code/client/clrcore/External/WeaponHash.cs

```diff
@@ -104,4 +104,34 @@ public enum VehicleWeaponHash
 		SearchLight = -844344963,
 		Radar = -764006018
 	}
-}
\ No newline at end of file
+
+	public enum AmmoType : ulong
+	{
+		Melee = 0,
+		FireExtinguisher = 1359393852,
+		Flare = 1808594799,
+		FlareGun = 1173416293,
+		PetrolCan = 18446744072810076321,
+		Shotgun = 18446744071831043387,
+		Pistol = 1950175060,
+		Ball = 18446744073702565478,
+		Sniper = 1285032059,
+		AssaultRifle = 218444191,
+		SMG = 1820140472,
+		Molotov = 1446246869,
+		StunGun = 18446744072370433504,
```

Comment:
Here as well. If only GH suggestions supported more than just one line in an obvious way...

```suggestion
		StunGun = 0xB02EADE0,
```